### ISSUE TITLE
Add Debian to distribution list for certificate installation

### ISSUE THREAD

**Issue by MrKrar** [craftcms/nitro#352]
### Description
Hello,
Using `nitro init` on Debian (WSL2) throws an error, the distribution seemingly not recognized for the certificate installation.
```
  … getting Nitro’s root site certificate ✓
Installing certificate (you might be prompted for your password)
Error: unable to find the distribution from the description: Description:       Debian GNU/Linux 10 (buster)
```
I found out the issue is similar to #323, but "Debian" seems to be missing in this [line](https://github.com/craftcms/nitro/blob/develop/pkg/certinstall/certinstall_linux.go#L102)?

**Comment by jasonmccallister** [comment]
Resolved this in https://github.com/craftcms/nitro/commit/0b6d2bf04e2ede714ccd00c8add78b16417a2caf for the next release.

**Comment by jasonmccallister** [comment]
Released in 2.0.8, let me know if you have any issues!
